README.md

```diff
@@ -1,13 +0,0 @@
-gr-cc11xx
-=========
-
-This GNU Radio out of tree module aims at providing a fully functional
```

Comment:
I'd not remove this file, the content is useful and is a a good entry page for github visitors. Alternative is to use the manifest above but this is not automatically picked by github I think

Reply:
Sorry, that was by mistake. Of course I added it.